src/api/test-helpers/loss-payee-financial-uk.ts

```diff
@@ -0,0 +1,24 @@
+import { ApplicationLossPayeeFinancialUk, TestHelperCreate } from '../types';
+
+/**
+ * create loss payee financial details uk helper
+ * Creates a blank LossPayeeFinancialUk.
+ * @param {Object} KeystoneJS context API
+ * @returns {Object} Created LossPayeeFinancialUk
+ */
+const createLossPayeeFinancialDetailsUk = async ({ context }: TestHelperCreate) => {
+  try {
+    console.info('Creating a financial details Uk (test helpers)');
+    const lossPayeeFinancialDetailsUk = (await context.query.LossPayeeFinancialUk.createOne({
+      data: {},
+      query: 'id',
+    })) as ApplicationLossPayeeFinancialUk;
+
+    return lossPayeeFinancialDetailsUk;
+  } catch (err) {
+    console.error(err);
```

Comment:
Add description to `console.error` please.

Reply:
this is a test helper btw.  Added